### DISCUSSION TITLE
Interactive DB exploration shell

### DISCUSSION THREAD

**Discussion by modolol** [MeteoSwiss/dvas#128] [Show and tell]
@fpavogt 
I made some changes in the way of working with the DB:
- Removed the context manager (it's frankly not clear the benefit of having one or not to read/write to the DB)
- Trying a utility to explore the DB
- ...a few things here and there that didn't seem to be very good....

Pull the branch db_access_mngmt, run the demo_script and tell me what you think... 😉

**Comment by fpavogt** [comment]
Wow ... this is very cool ... and very unexpected too ! The interactivity was a big surprise, but it certainly can be very handy to rapidly explore the content of the DB. However, is it also possible to call these `info`, `prm`, etc ... function from a script (i.e. non-interactively) ? This sort of scripted calls would be very useful to setup automated reduction recipes without having to know exactly (e.g.) the dates and/or amount and/or variables of launches in the DB. What do you think ?
